src/stylesheets/Login.scss

```diff
@@ -0,0 +1,23 @@
+#login {
+  margin-bottom: 15px;
+}
+
+.row {
+  .input-field {
+    input:focus:not {
+      border-bottom: 1px solid #ccc;
+    }
+    input:focus + input:not.invalid {
+      border-bottom: 1px solid #00bcd4 !important;
+    }
+    input:focus + label {
+      color: #00bcd4 !important;
```

Comment:
From my testing, it doesn't work if I don't use `!important`. For reference: http://stackoverflow.com/questions/37127123/change-color-of-underline-input-and-label-in-materialize-css-framework